gooddata-server-oauth2-autoconfigure/src/main/kotlin/CognitoLogoutHandler.kt

```diff
@@ -87,6 +90,7 @@ class CognitoLogoutHandler(
         UriComponentsBuilder
             .fromUri(endSessionEndpoint)
             .queryParam("client_id", clientId)
+            // https://docs.aws.amazon.com/cognito/latest/developerguide/logout-endpoint.html
```

Comment:
The URL is mentioned in the class-level KDoc, not sure if you find it convenient also here.

Reply:
Removed.